lib/model/list_model.dart

```diff
@@ -50,7 +50,8 @@ class ListModel {
   }
 
   Future<void> add(Item newItem) async {
-    newItem.order = itemCount;
+    // Use an order value larger than the largest order value.
+    newItem.order = (maxBy(items, (item) => item.order)?.order ?? -1) + 1;
```

Comment:
This little bit of logic makes sure the new item ALWAYS goes to the end of the ordering